### PR TITLE
Changing search params

### PR DESCRIPTION
**Pull request by megwat**
I changed the parameters in search to just account for user stories A3,4 and 5. I also changed this in the networking component. I changed the http request from POST to GET and changed the keys of the body to match the keys of the db and of the server.

### CODE REVIEW

**Review by elksy — APPROVED** [review]
Looks great

File: worldbank/src/Components/Networking.js [hunk 1]
```diff
@@ -47,7 +47,6 @@ class Networking {
         country: country,
         indicator: indicator,
         start: start,
-        end: end,
```
Comment (Omar-Shueb) [inline worldbank/src/Components/Networking.js:50]: Do you think we should send the data in the body rather than the header?

Comment (Omar-Shueb) [reply worldbank/src/Components/Networking.js:50]: nvm, I just realised this is a GET request so it has no body

File: worldbank/src/Components/Networking.js [hunk 2]
```diff
@@ -36,8 +36,8 @@ class Networking {
     return json;
   };
 
-  postSearch = async (country, indicator, start, end) => {
-    let response = await fetch(`${url}/createaccount`, {
+  postSearch = async (country, indicator, start) => {
+    let response = await fetch(`${url}/search`, {
       method: "POST",
```
Comment (Omar-Shueb) [inline worldbank/src/Components/Networking.js:41]: If its a GET request does this need to be changed then?
```suggestion
      method: "GET",
```

**Review by Omar-Shueb — APPROVED** [review]

**Pull request merged**